android/java/org/chromium/chrome/browser/vpn/billing/InAppPurchaseWrapper.java

```diff
@@ -150,9 +157,14 @@ private void endConnection() {
     public String getProductId(SubscriptionType subscriptionType) {
         boolean isReleaseBuild = ContextUtils.getApplicationContext().getPackageName().equals(
                 BraveConstants.BRAVE_PRODUCTION_PACKAGE_NAME);
```

Comment:
minor suggestion: I think `isReleaseBuild` is not being used anywhere

Reply:
oops. good catch. forgot to remove.